src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -152,6 +152,10 @@ def create_clone(self, pool, source_volname, source_subvolume, snapname):
             # attributes of subvolume's content though, are synced during the cloning process.
             attrs = source_subvolume.get_attrs(source_subvolume.snapshot_data_path(snapname))
 
+            # Set quota after the clone is successfull. Since quota is not very strictly enforced
+            # at byte level, it might cause clone to fail if quota is enforced before clone completion.
```

Comment:
"Since quota is not very strictly enforced at byte level, it might cause clone to fail if quota is enforced before clone completion" sentence is not clear to me. It doesn't clearly mention the issue. Can you reword it?   "Suggestion: The source of the clone may have exceeded its quota limit as CephFS quotas are imprecise. Cloning such a source may fail if the quota on the destination is set before starting the clone copy. So always set quota on destination after cloning is successful."

Reply:
Done